Include/funcobject.h

```diff
@@ -55,7 +55,6 @@ PyAPI_DATA(PyTypeObject) PyFunction_Type;
 #define PyFunction_Check(op) Py_IS_TYPE(op, &PyFunction_Type)
 
 PyAPI_FUNC(PyObject *) PyFunction_New(PyObject *, PyObject *);
-PyAPI_FUNC(PyObject *) PyFunction_NewWithQualName(PyObject *, PyObject *, PyObject *);
```

Comment:
Unfortunately removing this is not allowed per PEP 387

Reply:
:+1:  I have now restored it.